doc/GoogleTranslateConnector.md

```diff
@@ -0,0 +1,67 @@
+## Reshuffle Google Translate Connector
```

Comment:
This is not following the new header for connector doc. See Jira Readme for reference: https://github.com/reshufflehq/reshuffle-jira-connector/blob/master/README.md